src/Symfony/Bridge/PhpUnit/Legacy/ExpectDeprecationTrait.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Legacy;
+
+/**
+ * @internal use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait instead.
+ */
+trait ExpectDeprecationTrait
+{
+    /**
+     * @param string $message
+     */
+    public function expectDeprecation(): void
+    {
+        if (1 > func_num_args() || !\is_string($message = func_get_arg(0))) {
+            throw new \InvalidArgumentException(sprintf('The "%s()" method requires the string $message argument.', __FUNCTION__));
+        }
+
+        \Closure::bind(function (string $message): void {
+            $this->expectDeprecation($message);
+        }, $o = new class() {
+            use ExpectDeprecationTraitBeforeV8_4 {
```

Comment:
Can't we copy/paste the code instead?

Reply:
I guess we can keep it simple and juste copy the code if you want to.